app/Transformers/UserTransformer.php

```diff
@@ -40,6 +42,13 @@ public function transform($user)
             'updated_at' => $user->updated_at ? $user->updated_at->__toString() : null,
             'confirmedPrivacyPolicy' => $user->confirmedPrivacyPolicy,
             'confirmedMarketingUse' => $user->confirmedMarketingUse,
+            'monthlySchoolFee' => $user->monthlySchoolFee,
          ];
+
+        if (Auth::user()->id == $user->id || Auth::user()->hasRole('ADMIN')) {
```

Comment:
takže si to mám pýtať ako argument funkcie?  public function transform($user, $authUser) ?